framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerScenarioBuilderImpl.java

```diff
@@ -34,10 +34,8 @@
     private boolean deployPrometheus = false;
 
     public WorkbenchKieServerScenarioBuilderImpl() {
-        envVariables.put(OpenShiftTemplateConstants.KIE_SERVER_USER, DeploymentConstants.getKieServerUser());
-        envVariables.put(OpenShiftTemplateConstants.KIE_ADMIN_USER, DeploymentConstants.getWorkbenchUser());
-
-        envVariables.put(OpenShiftTemplateConstants.DEFAULT_PASSWORD, DeploymentConstants.getWorkbenchPassword());
+        envVariables.put(OpenShiftTemplateConstants.CREDENTIALS_SECRET, DeploymentConstants.getAppCredentialsSecretName());
+        envVariables.put(OpenShiftTemplateConstants.DEFAULT_PASSWORD, DeploymentConstants.getAppPassword());
```

Comment:
Why there is used both secret and password?

Reply:
I'm not sure about this. It was set as it is before and didn't want to change it... should we try to get rid of the DEFAULT_PASSWORD and try the test?

Reply:
What is the operator behaviour in case of this trial environment? Does it use secret or credentials from env variables?

Reply:
I've just learnt that the DEFAULT_PASSWORD is only used in the trial template. In this scenario, it will use the KIE_ADMIN_USER and DEFAULT_PASSWORD.
When the fix is working back, I will run some tests to ensure the trial template is working fine.
Thanks!

Reply:
ok

Reply:
I've tested some scenarios using this builder and as expected they were failing because unauthorised. I already fixed them by:
- re-adding the KIE_ADMIN_USER parameter (which is only used in the trial template)
- setting the KIE_ADMIN_USER and CREDENTIALS_SECRET both

I've running the whole suite for templates again and will update the PR summary with the latest results.

Reply:
So do we need there CREDENTIALS_SECRET ?

Reply:
I think we do. As far I know, this builder can be used for trial templates and also for the rest of templates. Therefore, we still need the SECRET_PASSWORD, and for the rest, the SECRET.
Let me know if this is correct.